packages/opds-web-client/src/interfaces.ts

```diff
@@ -14,7 +14,9 @@ export type MediaType =
   | "text/html;profile=http://librarysimplified.org/terms/profiles/streaming-media"
   | "application/audiobook+json"
   | "application/vnd.overdrive.circulation.api+json;profile=audiobook"
-  | "application/vnd.overdrive.circulation.api+json;profile=ebook";
+  | "application/vnd.overdrive.circulation.api+json;profile=ebook"
+  | "application/vnd.librarysimplified.web-epub"
+  | "application/vnd.librarysimplified.axisnow+json";
```

Comment:
I'm not totally certain that all of these should be added here since my understanding is that the `.web-epub` type is never returned from the CM, but is only visible and used within CPW. Can you describe for me how you're planning to implement this or maybe we can schedule a conversation to do that?

Reply:
Can you add me to the convo as well?

Reply:
Yep will do!

Reply:
For the test server I am using with circulation-patron-web the type of the exploded EPubs is `application/vnd.librarysimplified.web-epub` however, now I know circulation manager doesn't return that type but returns `application/vnd.librarysimplified.axisnow+json`. I will extend the `MediaType` directly in circulation-patron-web to include `"application/vnd.librarysimplified.web-epub"` if that ends up being necessary.